x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec

```diff
@@ -1819,3 +1819,12 @@ warning:Line 3:17: java.lang.ArithmeticException: / by zero
 w_avg:double
 null
 ;
+
+shadowingInternal
+FROM employees
+| STATS x = MAX(emp_no), x = MIN(emp_no)
```

Comment:
I don't think we have a more extreme test, but maybe it's worth adding:

`FROM employees | STATS x = MAX(salary), x = MIN(salary) by x = gender`

Reply:
Good one, definitely worth adding.